tests/logictest/suites/gen/00_dummy/00_0000_dummy_select_1

```diff
@@ -16,7 +16,7 @@ a
 statement query B label(http)
 SELECT NOT(1=1);
 
----- 
+----  
```

Comment:
why do we need these empty spaces?

Reply:
It cause by gen_suites.py scripts, three handler a little diferent with two handler when parsing result label.

Reply:
Ok, but my editor will always trim the trailing whitespace on save (most vscode users will have this settings by default).  I suggest to trim the spaces if label is empty.